### PR TITLE
Test user can add product to a customer's order

### PR DESCRIPTION
**Pull request by ghost**
Tests that a user can add a product to an active customer's order.

## Description
setUp and tearDown methods will run before and after each test.  These are necessary so that the order total is reset to zero after running OrderIncludesProductAdded to test that the order total is incremented by the price of the product in PriceOfProductAddedIsAddedToOrderTotal.

 Tests were dependent on Order and Product classes, which are included with basic getters and setters for writing the tests.  

## Related Issue
 Issue #7.  Also Issue #6.

## Motivation and Context
To add a product to the active customer's order, the superuser would need to confirm that the order is active and belongs to the active customer, and that once the product is added to the order that the order items include the product and the order total has increased by the price of the product.

## Steps to Test
```python TestAddProduct.py``` should return 4 passing tests.

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: TestAddProduct.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import unittest
+from customer import *
+from order import *
+from product import *
+
+class TestAddProduct(unittest.TestCase):
+
+    """
```
Comment (mccordgh) [inline TestAddProduct.py:8]: nice docstring 👍

File: TestAddProduct.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    @classmethod
+    def setUp(self):
+        self.bob = Customer("Albert", "Einstein","123 Atom Way",
+                            "Apt. B2", "Nashville", "TN",
+                            "32233", "615-555-555", "bigal@al.com")
+        self.new_product = Product('yoyo', 5)
+        self.active_order = Order(self.bob)
+
+    def tearDown(self):
```
Comment (mccordgh) [inline TestAddProduct.py]: need an `@classmethod` decorator above tearDown. ie:
```
@classmethod
def tearDown(self):
    self.active_order.total = 0
```

**Review by rtwhitfield84 — APPROVED** [review]
looks good 👍

**Review by mccordgh — CHANGES_REQUESTED** [review]
Tests passed, just a need a small syntax update.

**Review by mccordgh — APPROVED** [review]
👍

**Pull request merged**